client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -264,6 +269,9 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			const settingsFields = {
 				site: keys( settings ),
 			};
+			const path = getCurrentRoute( state )
+				.replace( getSiteSlug( state, siteId ), ':site' )
+				.replace( siteId, ':siteid' );
```

Comment:
I'll refer to [another comment](https://github.com/Automattic/wp-calypso/pull/32776#discussion_r280740352) about this code. We're now duplicating it in three different places, maybe it's time to do something about it.

Reply:
Will be done in a followup PR